VirtoCommerce.Storefront.Model/Security/Contracts/UpdateUserResponseDto.cs

```diff
@@ -0,0 +1,7 @@
+namespace VirtoCommerce.Storefront.Model.Security.Contracts
+{
+    public class UpdateUserResponseDto
```

Comment:
no need for dedicated ResponseDto class in here. Create and use more general SecurityResponseDto

Reply:
![image](https://user-images.githubusercontent.com/8361000/87168211-6e6e5f00-c2ce-11ea-8eb4-6f10d82a6ebe.png)

Reply:
yes, and there is alias:
![image](https://user-images.githubusercontent.com/5801549/87255040-f94a8780-c48f-11ea-9f19-d76288c80138.png)